src/chrome/content/rules/Nature.org.xml

```diff
@@ -0,0 +1,74 @@
+<!--
+	Bad certificate:
```

Comment:
It would be better to have more detailed information, like cert mismatch, expired, chain issues etc.